circuits/scripts/withdraw.js

```diff
@@ -0,0 +1,130 @@
+import { expect, assert } from 'chai';
+import { ProjectivePoint, signAsync } from '@noble/secp256k1';
+import { default as vmtree } from 'vmtree-sdk';
+
+import { MerkleTree as FixedMerkleTree } from "fixed-merkle-tree";
+
+class MerkleTree extends FixedMerkleTree {
+    constructor({ hasher, levels = 20, leaves = [], zero = 0 }) {
+        super(levels, leaves, {
+            hashFunction: (left, right) => hasher([left, right]),
+            zeroElement: zero,
+        });
+    };
+};
```

Comment:
We should put this in a shared file, since it's used in exactly the same way in `withdraw.js` and `mint.js`.